explainer.md

```diff
@@ -415,17 +415,21 @@ xrSession.addEventListener('inputsourceschange', (ev) => {
 
-An input source will also provide its preferred pointing ray, which is defined as a ray originating at `[0, 0, 0]` and extending down the negative Z axis, transformed bt the `pointerMatrix` attribute of an `XRInputPose` object. `pointerMatrix` will never be `null`. It's value will differ based on the type of input source that produces it, which is represented by the `pointerOrigin` attribute:
+An input source will also provide its preferred target ray on its pose, which is defined as a ray originating at `[0, 0, 0]` and extending down the negative Z axis, transformed by the `targetRayMatrix` attribute of an `XRInputPose` object. `pointerMatrix` will never be `null`. Their values will differ based on the type of input source that produces it, which is represented by the `targetRayMode` attribute:
 
-  * `'head'` indicates the pointer ray will originate at the user's head and follow the direction they are looking. (This is commonly referred to as a "gaze input" device.) There should be at most one `'head'` input source per session.
-  * `'hand'` indicates that the pointer ray originates from a handheld device, and represents the device's preferred targeting ray. The exact orientation of the ray relative to the device should follow platform-specific guidelines if there are any. In the absence of platform-specific guidance, the pointer ray should most likely point in the same direction as the user's index finger if it was outstretched while holding the controller.
-  * `'screen'` indicates that the input source was an interaction with the 2D canvas of a non-exclusive session, such as a mouse click or touch event. See [Magic Window Input](#magic_window_input) for more details.
+  * `'gazing'` indicates the target ray will originate at the user's head and follow the direction they are looking (this is commonly referred to as a "gaze input" device). While it may be possible for these devices to be tracked (and have a grip matrix), the head gaze is used for targeting. Example devices: 0DOF clicker, regular gamepad, voice command, tracked hands.
+  * `'pointing'` indicates that the target ray originates from a handheld device and represents that the user is using that device for pointing. The exact orientation of the ray relative to the device should follow platform-specific guidelines if there are any. In the absence of platform-specific guidance, the target ray should most likely point in the same direction as the user's index finger if it was outstretched while holding the controller.
+  * `'tapping'` indicates that the input source was an interaction with the 2D canvas of a non-exclusive session, such as a mouse click or touch event. See [Magic Window Input](#magic_window_input) for more details.
```

Comment:
Not sure if the term `tapping` feels accurate here. People typically associate it with touchscreens, and we want it to cover mouse and stylus input as well. Admittedly I don't have a better term in mind.

Reply:
This is a tough one to name; some other ideas:

- `projecting` - as in, the pointing ray is projected into the scene from the camera as defined by a point on the near plane
- `touching` - mouse, touch, stylus, etc could maybe considered forms of touch?
- `pressing` - same reasoning as touching

Reply:
`projecting` is probably the most accurate, but it doesn't seem intuitive. Maybe 'screen-projection' or something like that?

While it's not exactly more accurate than `tapping`, but the term `touching` feels more comfortable for whatever reason. At the very least tapping sounds like an instantaneous event while touching implicitly includes drag or long press operations.

Reply:
`screen-projection` seems to be the clearest, but `touching` feels more in line with the other enum values and is my preference by a narrow margin. Thoughts from other vendors welcome!